src/ng-annotate-main.js

```diff
@@ -1004,11 +1023,27 @@ function judgeInjectArraySuspect(node, ctx) {
 }
 
 function jumpOverIife(node) {
```

Comment:
The changes in this function look good (besides the comment). Never seen an IIFE using arrow functions, but I think it makes sense.